cpp-fs/storage/test/test_storage.cpp

```diff
@@ -0,0 +1,66 @@
+#include <gtest/gtest.h>
+
+#include "error_types.h"
+#include "partition/in_memory_partition.hpp"
+#include "storage.hpp"
+
+namespace tests::storage {
+
+namespace {
+template <typename T>
+  requires std::is_default_constructible_v<T>
+
+auto CreateDefaultStorage() {
+  return new cppfs::storage::Storage<T>(std::make_unique<T>());
+}
+}  // namespace
```

Comment:
actually all tests should be placed into the anonymous namespace

Reply:
my bad